cogs/countdown.py

```diff
@@ -25,11 +27,9 @@ def cog_unload(self):
 
     @tasks.loop(hours=24)
     async def announcement(self):
-        min_days = 42
         next_event = None
         for event in EVENTS:
-            days_until_event = event.days_until_event()
-            if days_until_event <= min_days:
+            if event.days_until_event() <= min(MIN_DAYS, next_event.days_until_event() if next_event is not None else MAX_DAYS):
```

Comment:
This logic actually doesn't make sense. It will always set it to the minimum days if the event is empty or the event is farther than 42 days out. Since 42 < 365 so the max days doesn't do anything. 

Simpler more readable logic
```python
days_until_next_event = MIN_DAYS
next_event = None

for event in EVENTS:
      if event.days_until_event() <=  days_until_next_event:
             next_event = event
             days_until_next_event = next_event.days_until_event()
```

Reply:
hey @keithrfung, logic can 100 percent be more readable and perhaps that's what garnered the comment _"This logic actually doesn't make sense."_ but unless i am too close to the code, if you replace `MAX_DAYS` with `MIN_DAYS`, you should see that the stack trace is nearly identical and both pieces of code return identical results. the main difference is that no space is explicitly allocated to store the number of days until the `next_event`. 

i truly appreciate the reviews you've given thus far; you're clearly a smart guy and i'm learning a lot from your feedback. however, i think that in the long run we can benefit from rewording statements like the one quoted above. it can come off a bit demoralizing and we don't want to scare off future contributors who might be newer to programming; i'm sure you had no ill intentions though <3 

updating with your version